apps/credential-showcase-api-server/src/database/repositories/ShowcaseRepository.ts

```diff
@@ -206,7 +209,7 @@ class ShowcaseRepository implements RepositoryDefinition<Showcase, NewShowcase>
       })
 
       return {
-        ...showcaseResult,
+        ...(showcaseResult as any), // TODO check this typing issue at a later point in time
```

Comment:
the only negative point is this cast as any, but I'll move this PR

Reply:
I've spent some time trying to fix it but it didn't work out, so I've just followed the pattern and appended a TODO to the code